transaction/core/src/ring_ct/rct_bulletproofs.rs

```diff
@@ -462,10 +501,68 @@ fn sign_with_balance_check<CSPRNG: RngCore + CryptoRng, S: RingSigner + ?Sized>(
     signer: &S,
     rng: &mut CSPRNG,
 ) -> Result<SignatureRctBulletproofs, Error> {
+    let ViewOnlySigningData {
+        extended_message: extended_message_digest,
```

Comment:
If this field contains the digest and not the actual message then it should be renamed to `extended_message_digest`.